docs/cookbook/recipe_sonata_admin_without_user_bundle.rst

```diff
@@ -28,35 +28,127 @@ This represents your security user, you can read more about it
 
     namespace App\Entity;
 
+    use App\Repository\UserRepository;
     use Doctrine\ORM\Mapping as ORM;
+    use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
     use Symfony\Component\Security\Core\User\UserInterface;
 
     /**
-     * @ORM\Entity
+     * @ORM\Entity(repositoryClass=UserRepository::class)
      */
-    final class User implements UserInterface
+    class User implements UserInterface, PasswordAuthenticatedUserInterface
```

Comment:
I think we should mention that this interface is only available for sf > 4.4 (not sure which minor o 5 introduced it, I guess it was the 5.4)

Reply:
Yes, we can mention it.

Reply:
I dont see the mention in the docs, did you add it?

Reply:
Introduce in 5.3, https://symfony.com/blog/new-in-symfony-5-3-improvements-for-security-users

@jobayerccj Can you add the mention ?